vita3k/modules/SceSysmem/SceSysmem.cpp

```diff
@@ -101,7 +101,7 @@ EXPORT(int, sceKernelFreeMemBlock, SceUID uid) {
 
     KernelState *const state = &host.kernel;
     const Blocks::const_iterator block = state->blocks.find(uid);
-    assert(block != state->blocks.end());
+    if (block == state->blocks.end()) return 0;
```

Comment:
```suggestion
    if (block == state->blocks.end())
        return 0;
```

When does this happen?

Reply:
i would like to ask you that question xD

Reply:
Yeah we can ignore this probably XD